File: core/src/main/java/com/bing/lan/core/base/mapper/IpLogMapper.java
package com.bing.lan.core.base.mapper;

import com.bing.lan.core.base.domain.IpLog;
import com.bing.lan.core.base.query.IpLogQueryObject;

import java.util.List;

public interface IpLogMapper {

    int deleteByPrimaryKey(Long id);

    int insert(IpLog record);

    IpLog selectByPrimaryKey(Long id);

    List<IpLog> selectAll();

    int updateByPrimaryKey(IpLog record);

    int queryForCount(IpLogQueryObject qo);

    List<IpLog> query(IpLogQueryObject qo);
}

File: core/src/main/kotlin/com/bing/lan/core/base/mapper/IpLogMapper.kt
package com.bing.lan.core.base.mapper

import com.bing.lan.core.base.domain.IpLog
import com.bing.lan.core.base.query.IpLogQueryObject

interface IpLogMapper {

    fun deleteByPrimaryKey(id: Long?): Int

    fun insert(record: IpLog): Int

    fun selectByPrimaryKey(id: Long?): IpLog

    fun selectAll(): List<IpLog>

    fun updateByPrimaryKey(record: IpLog): Int

    fun queryForCount(qo: IpLogQueryObject): Int

    fun query(qo: IpLogQueryObject): List<IpLog>
}